x-conduit/test/Test/X/Data/Conduit/Binary.hs

```diff
@@ -37,14 +38,26 @@ prop_slurp =
     forAll (choose (100, 4096) :: Gen Int) $ \buffer ->
       forAll (choose (0, length bs)) $ \offset ->
         forAll (choose (0, (length bs) - offset)) $ \chunk -> testIO $ do
-          withSystemTempDirectory "conduit" $ \p -> do
+          withTempDirectory "." "conduit" $ \p -> do
```

Comment:
See the "robust builds" guide in the engineering repo, makes it easier to deal with tmpdir space issues.